src/TimerUnit.cpp

```diff
@@ -177,7 +177,7 @@ TTimer* TimerUnit::getTimerPrivate(int id)
     if (mTimerMap.find(id) != mTimerMap.end()) {
         return mTimerMap.value(id);
     } else {
-        return 0;
+        return nullptr;
```

Comment:
This whole method seems pointless as it is identical to previous `TTimer* TimerUnit::getTimer(int)`...